src/vignette/storage/static_assets.clj

```diff
@@ -0,0 +1,48 @@
+(ns vignette.storage.static-assets
+  (:require [vignette.storage.protocols :refer :all]
+            [org.httpkit.client :as http]
```

Comment:
It could also work when reading from CEPH. We don't really use any of the s3 specific features. As far as I know it's just a `GET` on the object.